public_dropin_environments/java_codegen/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # This is the default base image for use with user models and workflows.
-FROM datarobot/java-dropin-env-base
+FROM datarobot/java-dropin-env-base:debian10-py3.7-java11
```

Comment:
Is it really debian OS? Have we tested it and certified it? Is it documented anywhere?

Reply:
We create this base so we know what is used. This is Debian

Reply:
Have you ever tested Debian? I've never had experience with it. How stable it is, etc.

Reply:
I haven't explicitly tested it. But we use it for legacy package images.
Quick google search says debian is more stable than ubuntu.